src/commands/networks/createNetwork.ts

```diff
@@ -0,0 +1,23 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { window } from 'vscode';
+import { InputBoxOptions } from 'vscode';
+import { IActionContext } from 'vscode-azureextensionui';
+import { ext } from '../../extensionVariables';
+import { wrapDockerodeENOENT } from '../../utils/wrapError';
+
+export async function createNetwork(_context: IActionContext): Promise<void> {
+
+    const opts: InputBoxOptions = {
+        value: '',
+        prompt: 'Name of the network'
+    };
+    const name = await ext.ui.showInputBox(opts)
```

Comment:
I think we should also prompt for [type](https://docs.docker.com/network/#network-drivers), i.e. host, bridge, etc. I'm not totally familiar with their different use cases, so we may not need to display all possible options.

Reply:
@philliphoff What do you think? Should we prompt at all?

Reply:
I think a prompt for the driver is probably a good idea, I'd be find to add it if people like the idea.

Reply:
In all of my cases, the default settings were sufficient (i.e. `docker network create <name>`) so I'm fine starting with that unless/until we have feedback where more control is desired.  Adding an option now to specify the driver would be totally fine, however.

Reply:
I've added a prompt for driver type in another commit.